.gitignore

```diff
@@ -0,0 +1,3 @@
+.vscode/
+coverage/
+node_modules/
```

Comment:
As I said in my last review, no files under the `dist/` directory should be checked into git and I shouldn't be seeing them in this PR. Add `dist/` to the .gitignore, then follow this tutorial to remove the JS files from git: https://www.codeblocq.com/2016/01/Untrack-files-already-added-to-git-repository-based-on-gitignore/